pkg/pillar/devicenetwork/pbr.go

```diff
@@ -88,12 +88,97 @@ func makeNetlinkRule(ifindex int, p net.IPNet, addForSubnet bool) *netlink.Rule
 	return r
 }
 
+func makeNetlinkGatewayRule(subnet net.IPNet, gateway net.IP, prio int) *netlink.Rule {
+	r := netlink.NewRule()
+	r.Table = syscall.RT_TABLE_LOCAL
+	r.Priority = prio
+	r.Family = HostFamily(gateway)
+
+	r.Src = &subnet
+	var g net.IPNet
+	g = HostSubnet(gateway)
+	r.Dst = &g
+
+	return r
+}
+
+func makeInwardNetlinkRule(ifindex int, p net.IPNet, addForSubnet bool, prio int) *netlink.Rule {
```

Comment:
This name is how it is used but not what it does.
Maybe naming it makeDstNetlinkRule and the original one makeSrcNetlinkRule would be more descriptive.
And makeNetlinkGatewayRule could be named makeDstLocalNetlinkRule since it adds rule towards local table?

Reply:
Ok. Will change the names.